plugin/secrets_service_account_key.go

```diff
@@ -217,20 +194,16 @@ func (b *backend) getSecretKey(ctx context.Context, s logical.Storage, rs *RoleS
 	resp.Secret.TTL = cfg.TTL
 
 	// If the request came with a TTL value, overwrite the config default
-	if ttl > 0 {
-		resp.Secret.TTL = time.Duration(ttl) * time.Second
+	if params.ttl > 0 {
+		resp.Secret.TTL = time.Duration(params.ttl) * time.Second
 	}
 
 	return resp, nil
 }
 
-const pathServiceAccountKeySyn = `Generate an service account private key under a specific role set.`
+const pathServiceAccountKeySyn = `Generate an service account private key secret.`
 const pathServiceAccountKeyDesc = `
-This path will generate a new service account private key for accessing GCP APIs.
-A role set, binding IAM roles to specific GCP resources, will be specified
-by name - for example, if this backend is mounted at "gcp", then "gcp/key/deploy"
-would generate service account keys for the "deploy" role set.
-
-On the backend, each roleset is associated with a service account under
-which secrets/keys are created.
+This path will generate a new service account key for accessing GCP APIs.
```

Comment:
I think we should also update `pathTokenHelpSyn` and `pathTokenHelpDesc`. It would be good to mention static accounts (similar to what's done here) and replace the example that's using the `gcp/token/<roleset>` API, since that's being marked as deprecated.

Reply:
Done in 533727eb13d6092015b229f1d87852c095ce0960